homeassistant/components/hassio/strings.json

```diff
@@ -413,5 +413,11 @@
         }
       }
     }
+  },
+  "config": {
+    "step": {},
+    "abort": {
+      "single_instance_allowed": "[%key:common::config_flow::abort::single_instance_allowed%]"
```

Comment:
If we declare it in the manifest it will use a common string and we don't need the string here.